public/js/reducers/chartsReducer.js

```diff
@@ -146,17 +93,13 @@ const chartsRedux = State({
                 isStacked: state.inWorkflowVsNotInWorkflow.isStacked
             }
         };  
-    },
-
-    getInWorkflowVsNotInWorkflowFailed(state, error) {
-        return {
-            ...state,
-            inWorkflowVsNotInWorkflow: {
-                ...state.inWorkflowVsNotInWorkflow,
-                error: error.message
-            }
-        };
     }
-});
+    case 'GET_IN_WORKFLOW_VS_NOT_IN_WORKFLOW_FAILED':
+        return Object.assign({}, state, { inWorkflowVsNotInWorkflow: { ...state.inWorkflowVsNotInWorkflow, error }});
```

Comment:
Do you need the state if you're erroring?

Reply:
Here I need the state because there is a single reducer for all the charts, and the errors are nested by chart, so if one chart has errors but the others don't, if I simply return `error` the state of all the charts will be affected. If I keep the state I can render the charts that fetched successfully and display errors for the one that failed